apps/central/src/components/Groups.vue

```diff
@@ -18,7 +18,21 @@
         placeholder="search in schemas"
         v-model="search"
       />
+
       <label>{{ count }} databases found</label>
+      <ButtonOutline
+        v-if="showChangeColumnButton && !showChangeColumn"
+        @click="showChangeColumn = true"
+        class="float-right"
+        >Show changelog</ButtonOutline
+      >
+      <ButtonOutline
+        v-if="showChangeColumnButton && showChangeColumn"
+        @click="showChangeColumn = false"
+        class="float-right"
+        >Hide changelog</ButtonOutline
+      >
```

Comment:
Layout of the <> brackets is ugly, could you change it to:
<tag>
  content
</tag>
?